dockerfiles/publish_manylinux/Dockerfile

```diff
@@ -0,0 +1,92 @@
+#Download base image pypa/manylinux2014_x86_64
+FROM quay.io/pypa/manylinux2014_x86_64:latest
+
+# Install wget
+RUN yum install wget atlas-devel \
+    lapack-devel blas-devel \
+    zlib libjpeg libpng \
+    libwebp libtiff \
+    -y && \
+    yum clean all
+# Install miniconda
+RUN wget -q \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/miniconda \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+
+# Prepare shared env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda activate base && \
+    pip install conan==1.51.1 && \
+    pip install twine && \
+    pip cache purge
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda activate base && \
+    conda install cmake && \
+    conda clean --all
+
+# gmake[2]: *** No rule to make target `/usr/local/lib/libtiff.so', needed by `lib/xrprimer_cpp.cpython-36m-x86_64-linux-gnu.so'.  Stop.
+RUN ln -s /opt/miniconda/bin/conan /opt/rh/devtoolset-10/root/usr/bin/conan && \
+    ln -s /opt/miniconda/bin/twine /opt/rh/devtoolset-10/root/usr/bin/twine && \
+    ln -s /opt/miniconda/bin/cmake /opt/rh/devtoolset-10/root/usr/bin/cmake && \
+    ln -s /usr/lib64/libjpeg.so.62 /usr/local/lib64/libjpeg.so && \
+    ln -s /usr/lib64/libwebp.so.4 /usr/local/lib/libwebp.so && \
+    ln -s /usr/lib64/libpng15.so.15 /usr/local/lib/libpng.so && \
+    ln -s /usr/lib64/libtiff.so.5 /usr/local/lib/libtiff.so
+
+# Update in bashrc
+RUN echo "source /opt/miniconda/etc/profile.d/conda.sh" >> /root/.bashrc && \
+    echo "conda deactivate" >> /root/.bashrc
+
+# Set proxy
+ENV HTTPS_PROXY http://172.16.1.135:3128
+ENV HTTP_PROXY http://172.16.1.135:3128
```

Comment:
Not necessary?

Reply:
Removed from dockerfile, but build history can be found at dockerhub:
https://hub.docker.com/layers/openxrlab/xrprimer_ci/manylinux2014_x86_64_torch1110_mmcv170/images/sha256-1a1b25dc3cefa40c832095dad4dc63bd9d4fd971d4467d5767e58352f4e439f1?context=repo